### PR TITLE
Add "SSO with Directus behind Proxy" guide

### PR DESCRIPTION
**Pull request by burka**
To support Directus running behind a proxy with SSO enabled you need to provide that configuration to access the SSO provider.

See https://github.com/directus/directus/discussions/11387 https://github.com/directus/directus/discussions/7790 https://github.com/directus/directus/pull/8223

Please correct anything that would improve this :-) I was just missing that bit of information in the official documentation :-)

### CODE REVIEW

**Comment by burka** [comment]
Fixed the formatting :-)

File: docs/self-hosted/sso.md [hunk 1]
```diff
@@ -192,3 +192,17 @@ The configuration disables secured cookies and should only be used in local envi
...
+
+If you are running the Directus server behind a HTTP(S)-Proxy to access your SSO provider you need to use global-agent
+for the proxy configuration, e.g.:
+
+```sh
+export GLOBAL_AGENT_HTTP_PROXY=${HTTP_PROXY} &&
+export GLOBAL_AGENT_HTTPS_PROXY=${HTTPS_PROXY} &&
+npx directus bootstrap &&
+node -r 'global-agent/bootstrap' node_modules/directus/dist/start.js
+```
```
Comment (br41nslug) [inline docs/self-hosted/sso.md:206]: Let's make sure to update this example to use docker instead of plain npm.

Comment (burka) [reply docs/self-hosted/sso.md:206]: Ok, I can adjust that tomorrow :-)

Comment (burka) [reply docs/self-hosted/sso.md]: Ok, I changed it to Docker  :-)

**Review by paescuj — APPROVED** [review]

**Comment by burka** [comment]
Awesome, thanks for the further improvements!

**Pull request merged**